projects/ion/src/lib/modal/modal.service.ts

```diff
@@ -21,6 +21,7 @@ import {
   providedIn: 'root',
 })
 export class IonModalService {
+  public readonly ionOnHeaderButtonAction = new Subject<SafeAny>();
```

Comment:
```suggestion
  public readonly ionOnHeaderButtonAction = new Subject<IonModalResponse | unknown>();
```
Same here.

Reply:
I think i just changed the types on the methods o.o